src/library/sidebarmodel.cpp

```diff
@@ -257,8 +258,17 @@ QVariant SidebarModel::data(const QModelIndex& index, int role) const {
             // If it's the "Quick Links" node, display it's name
             if (pTreeItem->getData().toString() == QUICK_LINK_NODE) {
                 return pTreeItem->getLabel();
+            } else {
+                if (!pTreeItem->getToolTip().isEmpty()) {
+                    return pTreeItem->getToolTip();
+                } else if (CmdlineArgs::Instance().getDeveloper()) {
+                    // Display the internal data only for debugging
+                    return pTreeItem->getData();
+                } else {
+                    // No tool tip
+                    return {};
+                }
```

Comment:
@uklotzde Sorry I'm looking into treeitem code just now. I would actually store the label as tooltip, but for that purpose we could simply return the label here (like for the QUICK_LINK_NODE) -- which would make the set/getTooltip methods obsolete...
What exactly do you have in mind with the tooltips?

Reply:
Each aoide entity (like a *playlist* or *collection*) has both a *label* and *notes* (= more verbose description). The *notes* are displayed as a tool tip.

Reply:
This also applies to the *prepared queries* that are stored locally.